tests/spotify_lights_sync_test.py

```diff
@@ -1,12 +1,11 @@
 from appdaemontestframework import automation_fixture
-import secrets
```

Comment:
From what I saw, this is not needed for running the test; which makes running tests on GitHub easier as their is no secrets to manage.

Reply:
You are right, this not needed. I will probably mock the spotipy component for future tests instead.